src/backend/oracle-data-api/src/main/java/ca/bc/gov/open/jag/tco/oracledataapi/model/EmailHistory.java

```diff
@@ -21,50 +26,79 @@
 @Setter
 @NoArgsConstructor
 public class EmailHistory extends Auditable<String> {
-	
+
 	/**
 	 * Primary key
 	 */
 	@Schema(description = "ID", accessMode = Schema.AccessMode.READ_ONLY)
 	@Id
 	@GeneratedValue
 	private Long emailHistoryId;
-	
+
 	/**
-	 * FromEmailAddress.
+	 * The date and time the email was sent
 	 */
-	@Column(length = 500)
+	@Column
+	@Temporal(TemporalType.TIMESTAMP)
 	@Schema(nullable = false)
```

Comment:
There is a disparity here I think. `@Column` nullable defaults to true, `@Schema` nullable defaults to false. One is true here and one is false. I think it's best to always specify nullable on both annotations and be explicit and not depend on defaults.

Reply:
That's a good point. Actually `@Column` annotation is only needed for H2 database but since we have not completely removed that, it would be better to be consistent.